templates/default/less/variables.less

```diff
@@ -62,6 +62,9 @@
 @font-size-h5:            ceil((@font-size-base * 1.1));
 @font-size-h6:            @font-size-base;
 
+// Controls font sizes in drop down menu.
```

Comment:
Use to Bootstrap convention to describe variables (see Services/UICore/lib/bootstrap-3.2.0/less/variables.less): 

//** Description of variable
@variable:               value;